.github/copilot-instructions.md

```
@@ -0,0 +1,704 @@
+| Endpoint Path                  | Router File                              |
+| ------------------------------ | ---------------------------------------- |
+| `/api/v1/client/agents/*`      | `app/api/v1/endpoints/agent/agent.py`    |
+| `/api/v1/client/attacks/*`     | `app/api/v1/endpoints/agent/attacks.py`  |
+| `/api/v1/client/tasks/*`       | `app/api/v1/endpoints/agent/tasks.py`    |
+| `/api/v1/client/crackers/*`    | `app/api/v1/endpoints/agent/crackers.py` |
+| `/api/v1/client/configuration` | `app/api/v1/endpoints/agent/general.py`  |
+| `/api/v1/web/*`                | `app/api/v1/endpoints/web/`              |
+| `/api/v1/control/*`            | `app/api/v1/endpoints/control/`          |
+| `/api/v2/client/agents/*`      | `app/api/v2/endpoints/agents.py`         |
+| `/api/v2/client/attacks/*`     | `app/api/v2/endpoints/attacks.py`        |
+| `/api/v2/client/tasks/*`       | `app/api/v2/endpoints/tasks.py`          |
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)